CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # cuML 0.8.0 (Date TBD)
 
 ## New Features
+- PR #636: Rand Index ml-prim
```

Comment:
Could we use the full word to describe the feature? `Random index metric ml-prim` is more understandable and still succinct

Reply:
sure @dantegd ! will make the change :D

Reply:
> Could we use the full word to describe the feature? `Random index metric ml-prim` is more understandable and still succinct
except, 'rand' is not short for 'random' but named after the dude who invented it 😅

Reply:
Lol my bad :)